templates/zerver/help/add-emoji.md

```diff
@@ -23,7 +22,7 @@ Zulip fully supports Unicode. To use Unicode in your message:
 
 ### Using emoji codes
 1. Start by composing a [new message or reply](/help/#posting-and-replying)
```

Comment:
This should be linked to the other **Replying to a message** doc we have instead of `(/help/#posting-and-replying)` since that might be removed.